src/queries/proposals.ts

```diff
@@ -114,3 +114,81 @@ export const getProposalById = (chainId: number, id: string): Promise<any> => {
     `;
   return processRequest(chainId, query);
 };
+
+export const getProposalsFromPlatformServices = (
+  chainId: number,
+  id: string,
+  timestamp?: string,
+): Promise<any> => {
+  const timestampCondition = timestamp ? `, updatedAt_gt: "${timestamp}"` : '';
+  const query = `
+      {
+        proposals(
+          orderBy: updatedAt
+          where: {status: Pending, service_: {platform: "${id}", status: Opened} ${timestampCondition}}
+        ) {
+          id
+          rateAmount
+          rateToken {
+            symbol
+          }
+          service {
+            id
+            buyer {
+              address
+            }
+            platform {
+              id
+              description {
+                website
+              }
+            }
+          }
+          seller {
+            address
+            handle
+          }
+        }
+      }
+    `;
+  return processRequest(chainId, query);
+};
+
+export const getAcceptedProposals = (
+  chainId: number,
+  id: string,
+  timestamp?: string,
+): Promise<any> => {
+  const timestampCondition = timestamp ? `, updatedAt_gt: "${timestamp}"` : '';
+  const query = `
+      {
+        proposals(
+          orderBy: updatedAt
+          where: {status: Validated, service_: {platform: "${id}"} ${timestampCondition}}
```

Comment:
must be like that as we need to the email from the platform where the seller his
```where: {status: Validated, platform: "${id}" ${timestampCondition}}```